### PR TITLE
Update work-with-cards.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Updates to card functionality: at-mentions, unfurl preview, calculations.

### CODE REVIEW

**Comment by justinegeffen** [comment]
@cwarnermm, I'd appreciate your honest feedback on this especially around context. Thank you! :)

**Comment by cwarnermm** [comment]
This new Boards page is a very good start, @justinegeffen. 

There are a number of things that users can do to customize the details captured in cards on a given board. Users would visit this page to learn about the many ways they can customize their own cards based on their specific needs. With this audience goal in mind, I'm envisioning the following:
- An introduction to what's possible; I recommend that the content start with simple concepts and build up to more complex concepts. 
- Detailed step-by-step instructions on how to complete key tasks.
- Thought leadership and guidance on how users can take full advantage of these card customization options based on common use cases explained through real-world examples that resonate.

As I reviewed this PR, I started asking myself a series of questions and used current Boards functionality to answer them:

What are the things you can do with cards?
- mention people
    - works the same as Channels
- share card previews
    - works the same as Channels (via link previews)
- customize card properties
    - create
    - property scope - affects all new/existing cards on the current board only
    - rename
    - delete
    - show/hide on a given board
    - anything else?
- work with property types
    - identify key use cases for each of the types based on real-world scenarios
- card calculations
    - unpack this step by step:
        - what properties and types do you need on a card to be able to take advantage of board calculation options?
        - where are calculation options available via a board?
        - how to use calculations?
        - real-world examples to demonstrate how calculations can be useful (based on the board's focus)

I had no idea how customizable the cards on a board could be! The more I learned and explored, the more I realized that there's SO MUCH MORE we could be doing with our own docs boards! This is the kind of lightbulb moment I want our readers to have too! To get them there, we need to showcase what's easy, what's possible, and what's awesome about our card customization options.

Here's what this could potentially look like:

**Mention people**
You can include a team member on a card by `mentioning them on a card <https://docs.mattermost.com/messaging/mentioning-teammates.html>`__ the same way you would in Channels. The team member you mention will receive a Direct Message notification from the Boards bot with a link to the card you mentioned them on. To mention multiple team members, separate each name with a comma. 

Mention auto-completes will be available in a future release.

[Note that I've started simple with user mentions, rather than diving right into the properties. This provides foundational grounding for the reader because user mentions in boards work exactly the same way as something they are likely already experienced with: Channels!]

**Share card previews**
When you share a link to a card within Mattermost, the card details are automatically displayed in a preview. This preview highlights what the card is about at a glance without having to click through to it.

[Similar philosophy here - a simple concept that ideally works the same as Channels via link previews; this helps users extend existing knowledge to new functional areas.]

**Customize card properties**
Cards can contain different data fields depending on the purpose of the board. For example, in a Roadmap board, cards include a Type field where you can add categories such as Bug, Triage, etc. In a Task board, cards include Status and Priority fields instead. 

**Add and manage properties**
On a card, you can create a new property field by selecting Add a property, then selecting the type of property from the drop-down menu. The property type specifies the type of data you plan to capture within that field. When you create new card properties, they are added to all new and all existing cards on the current board.

Properties are automatically added to the board filter list at the top of the page, so ensure you customize all property names to make it easy to filter your board by specific properties later.

You can rename a property at any time by opening a card, then selecting New Property. Your changes are saved immediate, and applied across all cards on the current board.

You can also delete properties you no longer need by selecting the property, then choosing Delete.  You'll be asked to confirm that you want to removes that property from every card on the current board. 

Once you have card properties defined, you have full control over which properties are visible on the board. Click Properties at the top of the board, then enable all properties you want to see at a glance, and hide all properties you don't want to see.

**Work with property types**
- checkboxes are useful for agenda items that are regularly revisited in weekly sprint or grooming meetings
- multi-select allows you to create badges to indicate things like status
- numbers are useful to capture metrics such as task sizing or effort estimates
- person provides a quick way to capture user assignments
- etc.

[Showcasing the power and awesomeness of the property types through real-world user applications is what's going to create that lightbulb moment.]

File: source/boards/work-with-cards.rst [hunk 1]
```diff
@@ -22,3 +22,53 @@ Cards can have different configuration fields, depending on the type of board yo
...
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+* **Checkboxes** are useful for agenda items that are regularly revisited in weekly sprint or grooming meetings.
+* **Multi-select** allows you to create badges to indicate things like status.
+* **Numbers** are useful to capture metrics such as task sizing or effort estimates.
+* **Person** provides a quick way to capture user assignments.
+
+Once you've added a property and named it, to display this information in the view you're using, select **Properties** and then select the property name.
+
+Calculations
```
Comment (justinegeffen) [inline source/boards/work-with-cards.rst:61]: @wuwinson: Some questions around calculations that I'm wondering if you can help with:

- What properties and types do you need on a card to be able to take advantage of board calculation options?
- What real-world examples do we have to demonstrate how calculations can be useful (based on the board's focus) other than the ones below? 

Thank you!

Comment (wuwinson) [reply source/boards/work-with-cards.rst:61]: @justinegeffen Played around with the calculations tool and documented my findings below. @harshilsharma63 @ogi-m @chenilim Can you please confirm if this is accurate:

1. **Count**: Counts the total number of rows in Table view or total number of cards in a column in Board view. Applies to any property type.
2. **Count Empty:** Applies to any property type
    1. Table View: Counts the total number of empty rows per column selected
    2. Board View: Counts the total number of empty values per property specified within the same column
3. **Count Not Empty**: Applies to any property type
    1. Table View: Counts the total number of rows with non-empty cells per column selected
    2. Board View: Counts the total number of non-empty values per property specified within the same column
4. **Percent Empty**: Applies to any property type
    1. Table View: Percentage of empty rows per column selected
    2. Board View: Percentage of empty values per property specified within the same column
5. **Percent Not Empty**: Applies to any property type
    1. Table View: Percentage of rows with non-empty cells per column selected
    2. Board View: Percentage of non-empty values per property specified within the same column
6. **Count Value**: Applies to any property type
    1. Table View: Counts the total number of values within the column (helpful for multi-select properties)
    2. Board View: Counts the total number of values per property specified within the same column.
7. **Count Unique Values**: Applies to any property type
    1. Table View: Counts the total number of rows with unique values within the column, omitting any duplicates from the count.
    2. Board View: Counts the total number of unique values per property specified within the same column, omitting any duplicates from the count.
8. **Sum**: The sum of any specified number property within the same column
9. **Average**: The average of any specified ****number property within the same column
10. **Median**: The median of any specified number property within the same column
11. **Min**: The lowest number of any specified number property within the same column
12. **Max**: The highest number of any specified number property within the same column
13. **Range**: Displays the lowest and highest number. Requires a number property.
14. **Earliest Date**: Displays the oldest date. Requires any custom date property or the included "Created time" or "Last updated time"
15. **Latest Date**: Displays the most recent date. Requires any custom date property or the included "Created time" or "Last updated time"
16. **Date Range**: The difference between the most recent date and oldest date within the same column. In Table View, it is labeled simply as "Range" for any date property/column. Requires any custom date property or the included "Created time" or "Last updated time"


Below are some other examples Chen provided in the v6.1 blog post on how calculations can be used:
- The count of cards where particular properties are empty (useful to make sure important info isn’t missing)
- The sum of estimated developer days for features (to make sure your team isn’t overloaded), and
- The range of estimated dates (to make sure your milestones all line up)

File: source/boards/work-with-cards.rst [hunk 2]
```diff
@@ -20,5 +20,97 @@ Work with Cards
...
 
-The default name for a new property is **New Property**. Ensure that you name your property as it's added to the filter list at the top of the board and having all your filters named **New Property** makes it difficult to work out what you're filtering. You can rename a property field by opening up a card and selecting **New Property**. Enter the new name in the field provided. The change is saved immediately and applied across all cards.
+The default name for a new property is **New Property**. Ensure that you name your property as it's added to the filter list at the top of the board and having all your filters named **New Property** makes it difficult to work out what you're filtering. To rename a property field, open up a card and select **New Property**. Enter the new name in the field provided. The change is saved immediately and applied across all cards.
```
Comment (cwarnermm) [inline source/boards/work-with-cards.rst:23]: It looks like the default name changes per property.

File: source/boards/work-with-cards.rst [hunk 3]
```diff
@@ -20,5 +20,97 @@ Work with Cards
...
 
+Mention people
+--------------
+
+You can include a team member on a card by mentioning them on a card <https://docs.mattermost.com/messaging/mentioning-teammates.html>__ the same way you would in Channels. The team member you mention will receive a Direct Message notification from the Boards bot with a link to the card you mentioned them on. To mention multiple team members, separate each name with a comma.
```
Comment (cwarnermm) [inline source/boards/work-with-cards.rst:28]:
```suggestion
You can include a team member on a card by `mentioning them on a card <https://docs.mattermost.com/messaging/mentioning-teammates.html>`__ the same way you would in Channels. The team member you mention will receive a Direct Message notification from the Boards bot with a link to the card you mentioned them on. To mention multiple team members, separate each name with a comma.
```

File: source/boards/work-with-cards.rst [hunk 4]
```diff
@@ -20,5 +20,97 @@ Work with Cards
...
+------------
+
+When you view a board in table view, you can use calculations to answer basic metric questions without needing to create complex reports. Hover over the bottom of a column to display the **Calculate** feature, then select the arrow to open the menu options.
+
+You can use calculations to quickly see:
+
+- How many story points are planned for a release.
+- How many tasks have been assigned or not assigned.
+- How long has the oldest bug been sitting in the backlog.
+- The count of cards where particular properties are empty (useful to make sure important info isn’t missing)
```
Comment (cwarnermm) [inline source/boards/work-with-cards.rst:71]:
```suggestion
- The count of cards where particular properties are empty (useful to make sure important info isn’t missing).
```

File: source/boards/work-with-cards.rst [hunk 5]
```diff
@@ -20,5 +20,97 @@ Work with Cards
...
+- The sum of estimated developer days for features (to make sure your team isn’t overloaded), and
```
Comment (cwarnermm) [inline source/boards/work-with-cards.rst:72]:
```suggestion
- The sum of estimated developer days for features (to make sure your team isn’t overloaded).
```

File: source/boards/work-with-cards.rst [hunk 6]
```diff
@@ -20,5 +20,97 @@ Work with Cards
...
+- The range of estimated dates (to make sure your milestones all line up)
```
Comment (cwarnermm) [inline source/boards/work-with-cards.rst:73]:
```suggestion
- The range of estimated dates (to make sure your milestones all line up).
```

File: source/boards/work-with-cards.rst [hunk 7]
```diff
@@ -20,5 +20,97 @@ Work with Cards
...
+
+Share card previews
+-------------------
+
+When you share a link to a card within Mattermost, the card details are automatically displayed in a preview. This preview highlights what the card is about at a glance without having to navigate to it.
+
+Customize card properties
+-------------------------
+
+Cards can contain different data fields depending on the purpose of the board. For example, in a Roadmap board, cards include a Type field where you can add categories such as Bug, Triage, etc. In a Task board, cards include Status and Priority fields instead.
```
Comment (cwarnermm) [inline source/boards/work-with-cards.rst:38]: Question: Do we want to consistently bold references to fields in Boards documentation?

**Review by cwarnermm — COMMENTED** [review]
Feedback provided inline. This topic is really coming together! Well done!

File: source/boards/work-with-cards.rst (hunk 7 above)
Comment (wuwinson) [inline source/boards/work-with-cards.rst:38]: See first comment up top.

File: source/boards/work-with-cards.rst [hunk 8]
```diff
@@ -20,5 +20,97 @@ Work with Cards
...
+
+Add and manage properties
+-------------------------
+
+To create a new a new property field open a card and select **Add a property**. Then select the type of property from the drop-down menu. The property type specifies the type of data you plan to capture within that field. When you create new card properties, they are added to all new and all existing cards on the current board.
+
+Properties are automatically added to the board filter list at the top of the page, so ensure you customize all property names to make it easy to filter your board by specific properties later. You can rename a property at any time by opening a card, then selecting **New Property**. Your changes are saved immediately, and applied across all cards on the current board.
```
Comment (wuwinson) [inline source/boards/work-with-cards.rst:45]: You can rename a property at any time by opening a card, then clicking on the **property name** to open an editable field.

File: source/boards/work-with-cards.rst (hunk 7 above)
Comment (justinegeffen) [reply source/boards/work-with-cards.rst:38]: Yes, I think so. Good catch, thanks!

File: source/boards/work-with-cards.rst [hunk 9]
```diff
@@ -20,5 +20,97 @@ Work with Cards
...
+
+Share card previews
+-------------------
+
+When you share a link to a card within Mattermost, the card details are automatically displayed in a preview. This preview highlights what the card is about at a glance without having to navigate to it.
+
+Customize card properties
+-------------------------
+
+Cards can contain different data fields depending on the purpose of the board. For example, in a Roadmap board, cards include a Type field where you can add categories such as Bug, Triage, etc. In a Task board, cards include Status and Priority fields instead.
```
Comment (justinegeffen) [inline source/boards/work-with-cards.rst:38]:
```suggestion
Cards can contain different data fields depending on the purpose of the board. For example, in a **Roadmap** board, cards include a **Type** field where you can add categories such as **Bug**, **Triage**, etc. In a **Task** board, cards include **Status** and **Priority** fields instead.
```

File: source/boards/work-with-cards.rst [hunk 10]
```diff
@@ -20,5 +20,97 @@ Work with Cards
...
+
+Add and manage properties
+-------------------------
+
+To create a new a new property field open a card and select **Add a property**. Then select the type of property from the drop-down menu. The property type specifies the type of data you plan to capture within that field. When you create new card properties, they are added to all new and all existing cards on the current board.
+
+Properties are automatically added to the board filter list at the top of the page, so ensure you customize all property names to make it easy to filter your board by specific properties later. You can rename a property at any time by opening a card, then selecting **New Property**. Your changes are saved immediately, and applied across all cards on the current board.
```
Comment (justinegeffen) [inline source/boards/work-with-cards.rst:45]:
```suggestion
Properties are automatically added to the board filter list at the top of the page, so ensure you customize all property names to make it easy to filter your board by specific properties later. To rename a property, open a card and select the property name to open an editable field. Your changes are saved immediately, and applied across all cards on the current board.
```

File: source/boards/work-with-cards.rst (hunk 2 above)
Comment (justinegeffen) [inline source/boards/work-with-cards.rst:23]:
```suggestion
The default name for a new property is the name of the property type (e.g. **Date**, **URL**) and properties are automatically added to the filter list at the top of the board. You can use this to toggle which properties you see displayed on each card preview. 

To rename a property field, open up a card and select **New Property**. Enter the new name in the field provided. The change is saved immediately and applied across all cards.
```

File: source/boards/work-with-cards.rst [hunk 11]
```diff
@@ -20,5 +20,99 @@ Work with Cards
 
 Cards can have different configuration fields, depending on the type of board you've selected. For example, in a Roadmap board you'll find the cards include an option for **Type** where you can add categories such as Bug, Triage, etc. A Task board doesn't include a Type field, but includes **Status** and **Priority**. All cards have a **New Property** field. For this field, you can select the type from the drop-down menu. It'll then be added to your card.
```
Comment (justinegeffen) [inline source/boards/work-with-cards.rst:21]:
```suggestion
Our templates provide some default card properties that can be customized or removed. In the Roadmap template, we include **Type** property, whereas in the Project Tasks template, we include a **Status** and **Priority** property. These properties are not exclusive to any template and can be easily re-created in any of the templates provided.


```

File: source/boards/work-with-cards.rst (hunk 1 above)
Comment (harshilsharma63) [reply source/boards/work-with-cards.rst:63]: @wuwinson that's absolutely correct! Sorry for missing this notification.

File: source/boards/work-with-cards.rst [hunk 12]
```diff
@@ -18,7 +18,101 @@ Work with Cards
...
+* **Multi-select** allows you to create badges to indicate things like status.
+* **Numbers** are useful to capture metrics such as task sizing or effort estimates.
+* **Person** provides a quick way to capture user assignments.
+* **URL** can be used to provide a link to a pull request or relevant website.
+* **Created time/Created by/Last updated time/Last updated by** are good ways order tasks in order of recency.
+
+Calculations
+------------
+
+When you view a board in table view, you can use calculations to answer basic metric questions without needing to create complex reports. Hover over the bottom of a column to display the **Calculate** feature, then select the arrow to open the menu options.
```
Comment (harshilsharma63) [inline source/boards/work-with-cards.rst:66]: > When you view a board in table view, you can use calculations to answer basic metric questions

This sentence hints that the `Calculations` feature is for table view only, but it's also for Kanban view/Board view as we've already mentioned in the docs below.

**Review by harshilsharma63 — CHANGES_REQUESTED** [review]

File: source/boards/work-with-cards.rst (hunk 12 above)
Comment (justinegeffen) [inline source/boards/work-with-cards.rst:66]:
```suggestion
When you view a board in table, Kanban, or Board view, you can use calculations to answer basic metric questions without needing to create complex reports. Hover over the bottom of a column to display the **Calculate** feature, then select the arrow to open the menu options.
```

**Review by harshilsharma63 — APPROVED** [review]

**Pull request merged**